src/lib/NativeModules/GraphQLQueryCache.ts

```diff
@@ -0,0 +1,18 @@
+import { NativeModules } from "react-native"
+const { ARGraphQLQueryCache } = NativeModules
+
+export function set(queryID: string, variables: object, response: string, ttl: number = 0): void {
+  ARGraphQLQueryCache._setResponseForQueryIDWithVariables(response, queryID, variables, ttl)
```

Comment:
Curious, why the `_`? Wouldn't this reaching in to the native side of things be considered public?

Reply:
I'd bet this is an indication of "you shouldn't use this" - we have to expose it, but that doesn't mean we have to let people consider it a public API for using inside RN components

Guess we shouldn't [do this](https://github.com/facebook/react/blame/d8ee071c92994879fa82ba4d2775b4f38fb78f1d/src/React.js#L25-L26)

Reply:
Yeah that native API is considered private and JS code should only use it through this JS module. However, tbh the original change was promoted because the native class needed to expose a public `clearAll` objc method as well that internally needed to dispatch to the internal GCD queue and that method would clash with the `clearAll` that gets exposed to the RN bridge.